src/Hl7.Fhir.STU3/Specification/Snapshot/SnapshotGeneratorExtensions.cs

```diff
@@ -73,6 +72,47 @@ public static void RemoveAllConstrainedByDiffExtensions<T>(this IEnumerable<T> e
             }
         }
 
+        internal static void RemoveAllNonInheritableExtensions(this Element element)
```

Comment:
Can we use a bit code in a shared project instead of copy/paste? I know the snapget is unique to both, but maybe this overlaps?

Reply:
I moved the extensions to base.

Reply:
I'm working on supporting extensions on element definnition properties in Forge. That would mean the UI won't be cluttered with these I guess? That would be nice.
![image](https://github.com/user-attachments/assets/589adfe0-8138-42ee-911f-e32578b77188)